package.json

```diff
@@ -19,7 +19,8 @@
     "afloop": "^1.0.1",
     "expect.js": "^0.3.1",
     "keyb": "^1.0.4",
-    "pixi.js": "^3.0.9"
+    "pixi.js": "^3.0.9",
+    "jquery": "latest"
```

Comment:
I'm always wary of using the latest.. that means they can break our code if the new version changes the API. Could we freeze this to a specific semantic version?